tests/mysql-command/test.yaml

```diff
@@ -0,0 +1,11 @@
+args:
```

Comment:
Could you please add min-version: 8 in the requires section as to have CI green for main7 branch ?